src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDataSource.java

```diff
@@ -663,6 +663,24 @@ public int getQueryTimeout() {
                 SQLServerDriverIntProperty.QUERY_TIMEOUT.getDefaultValue());
     }
 
+    public void setEnablePrepareOnFirstPreparedStatementCall(boolean enablePrepareOnFirstPreparedStatementCall) {
+        setBooleanProperty(connectionProps, SQLServerDriverBooleanProperty.ENABLE_PREPARE_ON_FIRST_PREPARED_STATEMENT.toString(), enablePrepareOnFirstPreparedStatementCall);
+    }
+
```

Comment:
Add javadocs to the new public APIs (we have missed them for most of the APIs in this class, we will add them sometime soon). If you are using eclipse IDE, typing /** (enter) will generate the basic javadocs, you can add details to it (example SQLServerDataSource#getServerNameAsACE).